aspenTools/python/source/aspen/blender/common/results_log/ui.py

```
@@ -0,0 +1,130 @@
+
+class ResultLogHandler(logging.Handler):
+    def __init__(self, window: ResultLogMainWindow, level: int = logging.NOTSET):
+        logging.Handler.__init__(self, level)
+        self.window = window
+
+    def emit(self, record: logging.LogRecord):
+        """ Overrides base class to call ResultLogMainWindow's print_log()
+        Args:
+            record (logging.LogRecord): Holds information about a given log.
+        """
+        msg = self.format(record)
```

Ruff (0.14.5)

Local variable `msg` is assigned to but never used

Remove assignment to unused variable `msg`

(F841)